mlflow/tensorflow.py

```diff
@@ -651,10 +656,6 @@ class __MLflowTfKerasCallback(Callback):
         Records model structural information as params after training finishes.
         """
 
-        def __init__(self, log_models):
-            super().__init__()
```

Comment:
Note: Since `log_models` is not added to `self`  in `__init__` anymore, removing the entire method so that super's init is invoked automatically without running into `Useless super delegation in method` linting error.